Author: <gregor+debian@comodo.priv.at>
Description: fix problems with dates in XML::Feed:Atom;
Cf. #380498
--- a/lib/XML/Feed/Format/Atom.pm
+++ b/lib/XML/Feed/Format/Atom.pm
@@ -260,7 +260,9 @@
     if (@_) {
         $entry->{entry}->issued(DateTime::Format::W3CDTF->format_datetime($_[0])) if $_[0];
     } else {
-        $entry->{entry}->issued ? iso2dt($entry->{entry}->issued) : undef;
+        return iso2dt($entry->{entry}->issued) if $entry->{entry}->issued;
+        return iso2dt($entry->{entry}->published) if $entry->{entry}->published;
+        return undef;
     }
 }
 
